docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+postgres:
+  image: postgres:latest
```

Comment:
Can it be `latest`? Should be stick to a version? cc @jroper

Reply:
I don't think it matters. The features of Postgres we're using haven't changed in over 20 years.